Add remove endpoint

NoK1R1LL opened the pull request:
Функциональность «Удаление фильмов и пользователей»

Alexander-Bolotov commented on src/main/java/ru/yandex/practicum/filmorate/service/FilmService.java:
```diff
@@ -0,0 +1,92 @@
+    }
+
+    public void addLike(Integer filmId, Integer userId) {
+        if (userStorage.getUserById(userId).isPresent() && filmStorage.getFilmById(filmId).isPresent()) {
```
Сделай в хранилище методы типа isExist которые будут говорить существует или нет запись, что бы не тащить ее из хранилища

NoK1R1LL committed:
Функциональность «Удаление фильмов и пользователей»

NoK1R1LL committed:
Функциональность «Общие фильмы»

NoK1R1LL committed:
Реализовать запрос в бд из FilmDbStorage

NoK1R1LL committed:
Завести метод в FilmService -> FilmServiceImpl который отправляет команду FilmDbStorage на удаление фильма.

NoK1R1LL committed:
Создать в FilmController метод, который будет работать с эндпоинтом DELETE /films/{filmId}.

NoK1R1LL committed:
Реализовать тесты в FilmDbTest

NoK1R1LL committed:
Реализовать запрос в бд

NoK1R1LL committed:
Завести метод в UserService -> UserServiceImpl

NoK1R1LL committed:
Создать в UserController метод, который будет работать с эндпоинтом DELETE /users/{userId}.

gorelov-m-v committed:
Разрешили конфликты при обновлении из develop

# Conflicts:
#	src/main/java/ru/yandex/practicum/filmorate/controller/UserController.java
#	src/main/java/ru/yandex/practicum/filmorate/service/user/UserService.java
#	src/main/java/ru/yandex/practicum/filmorate/service/user/UserServiceImpl.java
#	src/main/java/ru/yandex/practicum/filmorate/storage/film/FilmDbStorage.java
#	src/main/java/ru/yandex/practicum/filmorate/storage/user/UserDbStorage.java
#	src/main/java/ru/yandex/practicum/filmorate/storage/user/UserStorage.java
#	src/test/java/ru/yandex/practicum/filmorate/db/UserDbTest.java

gorelov-m-v committed:
fix -кодстайл
